docs/compatibility.md

```diff
@@ -16,6 +16,10 @@ guarantee it.
 
 We're eager to review pull requests from our Windows user base!
 
+## Python
+
+pygradle currently supports Python 2.7, and 3.4 - 3.7.
```

Comment:
3.5 - 3.7 perhaps?
Need to make sure we're not running 3.4 in some Travis or Circle CI setting though.

Reply:
I looked through `.travis.yml` and `.circleci/config.yml` and didn't see anything obvious, so unless that's hidden somewhere not in the code, I don't think it is.

I bumped the version number to 0.9.0